test/controllers/private_messages_controller_test.rb

```diff
@@ -0,0 +1,19 @@
+require "test_helper"
+
+class PrivateMessagesControllerTest < ActionController::TestCase
+  setup do
+    @ticket = tickets(:problem)
+    @reply = replies(:solution)
+
+    sign_in users(:alice)
+  end
+
+  test "create private message" do
+    put :create, private_message: {
+                   ticket_id: @ticket.id,
+                   message: "test message"
+               }
```

Comment:
Indent the right brace the same as the start of the line where the left brace is.